tests/blog/tests_index.py

```diff
@@ -14,10 +14,10 @@ class BlogPage(TestCase):
     render_templates = False
 
     def create_app(self):
-        my_app = app
-        my_app.testing = True
+        app = create_app(testing=True)
+        app.testing = True
```

Comment:
Would setting `app.testing` inside the `create_app(testing=True)` make more sense?